azure-pipelines.yml

```diff
@@ -23,7 +23,21 @@ pr:
 - production
 
 stages:
+- stage: GitHub_Issue_Verification
+  displayName: GitHub Issue Verification
+  condition: and(eq(variables['Build.Reason'], 'PullRequest'), ne(variables['Build.SourceBranch'], 'refs/heads/production'))
```

Comment:
Won't this break in AzDO runs? Should we rather check for "runs in the public project"?